x-pack/metricbeat/module/meraki/device_uplink_loss_and_latency/_meta/data.json

```diff
@@ -0,0 +1,19 @@
+{
+    "@timestamp":"2016-05-23T08:05:34.853Z",
+    "beat":{
+        "hostname":"beathost",
+        "name":"beathost"
+    },
+    "metricset":{
+        "host":"localhost",
+        "module":"meraki",
+        "name":"device_uplink_loss_and_latency",
+        "rtt":44269
+    },
+    "meraki":{
+        "device_uplink_loss_and_latency":{
+            "example": "device_uplink_loss_and_latency"
```

Comment:
I see for almost all the metricsets we are getting a single field (example). 
I hope that is what is expected through these metricsets ?